docs/user-guide/src/bmo/live-iso.md

```diff
@@ -0,0 +1,32 @@
+# Live-ISO
+
+The live-iso API in Metal3 allows to boot BaremetalHost with live-iso image instead of using IPA deploy ramdisk.
```

Comment:
When used in a normal text, it should be "live ISO" or "live CD", not "live-iso". So for example, "with a live ISO image.."

Reply:
Fixed in normal text usage.